Jenkinsfiles/Jenkinsfile.deploy_ami

```diff
@@ -172,6 +224,42 @@ pipeline {
         }
       }
     }
+
+    stage('Django - migrate and collectstatic') {
```

Comment:
So, `collectstatic` is fairly innocuous in my experience. It merely copies static files to the CDN, doesn't have any effect on the health of the application.

For `migrate`, there may be some additional planning necessary to ensure the hosts can launch an register as healthy. For example, a migration may require two deployments. There may be a initial release that simply adds database fields, but does not use said fields in the application, since referring to them _before_ they are present would almost certainly cause the application to fail. After the migration release, a subsequent release with references to the new fields can be deployed. 

Both of these commands are configured to run on only _one_ host and only _once_ per deployment. They do not need to be rerun as new instances are introduced to the group.